tests/test_serializers.py

```diff
@@ -0,0 +1,142 @@
+from mock import Mock
+
+from tests import SerializerTestCase, TestPreference, TestUser
+from madprops.serializers import PropertiesSerializer
+
+
+class TestSerializer(PropertiesSerializer):
+    class Meta:
+        model = TestPreference
+        parent_obj_name = 'user'
+        read_only_props = ('user_token',)
+
+
+class FieldToRepresentation(SerializerTestCase):
+    def setUp(self):
+        self.preferences = {'k1': 'v1', 'k2': 'v2'}
+        user = TestUser()
+        user.preferences = self.preferences
+        self.representation = TestSerializer().field_to_native(
+            user, 'preferences')
+
+    def test_converts_properties_to_dict(self):
+        self.assertEqual(self.preferences, self.representation)
+
+
+class PropertyToInternalValue(SerializerTestCase):
```

Comment:
Sorry, I used terminology from DRF 3.0 (which makes more sense to me vs "from/to-native").  But you're right.  Probably (since this lib is for DRF < 3.0) we should use from/to native.